docs/src/main/sphinx/installation/kubernetes.rst

```diff
@@ -0,0 +1,195 @@
+===================
+Trino on Kubernetes
+===================
+
+`Kubernetes <https://kubernetes.io>`_ is a container orchestration platform that
+allows you to deploy Trino and other applications in a repeatable manner across
+different types of infrastructure. This can range from deploying on your laptop
+using tools like `KIND <https://kind.sigs.k8s.io>`_ and
+`Helm <https://helm.sh>`_, to running on a managed Kubernetes service on the
+major clouds like `Amazon EKS <https://aws.amazon.com/eks>`_,
+`Google GKE <https://cloud.google.com/kubernetes-engine>`_, and
+`Azure AKS <https://azure.microsoft.com/en-us/services/kubernetes-service>`_.
+This is valuable in that it allows you to prototype on you local or on-premise
+cluster and use the same deployment mechanism to deploy to the cloud to scale up
+your deployment.
+
+Prerequisites
+-------------
+* A running Kubernetes cluster
+    * For local deployments, we recommend using `KIND <https://kind.sigs.k8s.io>`_
+* `kubectl <https://kubernetes.io/docs/tasks/tools/#kubectl>`_
+* `helm <https://helm.sh>`_
+
+
+Running Trino using Helm
+------------------------
+
+The fastest way to run Trino on Kubernetes is to use the
+`Trino Helm chart <https://github.com/trinodb/charts>`_. Helm is a package
+manager for Kubernetes applications that allows for simpler installation and
+versioning by templating Kubernetes configuration files. Run the following
+commands from the system with ``helm`` and ``kubectl`` installed, that points to
+your running Kubernetes cluster:
+
+#. Add the Trino Helm chart repository to Helm. This tells Helm where to find
+   the Trino charts. You can name the repo whatever you want but we will use
+   ``trino``.
+
+   .. code-block:: text
+
+       helm repo add trino https://trinodb.github.io/charts
+
+#. Install the Trino helm chart. This fetches the template and uses the local
+   ``values.yaml`` file to populate the templates to generate the Kubernetes
+   configuration files.
+
+   .. code-block:: text
+
+       helm install my-trino-cluster trino/trino
+
+   You should see output as follows:
+
+   .. code-block:: text
+
+       NAME: my-trino-cluster
+       LAST DEPLOYED: Tue Sep 13 14:12:09 2022
+       NAMESPACE: default
+       STATUS: deployed
+       REVISION: 1
+       TEST SUITE: None
+       NOTES:
+       Get the application URL by running these commands:
+         export POD_NAME=$(kubectl get pods --namespace default -l "app=trino,release=my-trino-cluster,component=coordinator" -o jsonpath="{.items[0].metadata.name}")
+         echo "Visit http://127.0.0.1:8080 to use your application"
+         kubectl port-forward $POD_NAME 8080:8080
+
+   This output will differ based on your ``values.yaml`` configuration and
+   cluster name. For example, the port ``8080`` is set by the
```

Comment:
Is it worth noting that this default Trino installation is not secured with any authentication, but also does not install any ingress rule that would expose it?

Reply:
Gonna default to @mosabua on this one. I think it wouldn't hurt, but I also don't want to drown anyone who wants to jump in with too much info.